understanding/20/labels-or-instructions.html

```diff
@@ -39,6 +39,9 @@ <h2>Intent of Labels or Instructions</h2>
          The goal is to make certain that enough information is provided for the user to accomplish
          the task without undue confusion or navigation.
       </p>
+
+      <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive labels without additional text. 
+        In these cases, authors should ensure that the image and its use as a label (in context) are widely understood.</p>
```

Comment:
Suggest merging the two sentences just into one?

```suggestion
      <p>Images (with an appropriate text alternative) can serve as descriptive labels without additional text. In these cases, authors should ensure that the image and its use as a label (in context) are widely understood.</p>
```

Reply:
I prefer to point out "Labels of form controls are usually text-based.".
It makes clear that using labels is the exception, and that we are talking about forms here.